apps/question-bank/src/test/resources/features/valueset/ValueSetSearch.feature

```diff
@@ -1,20 +1,19 @@
-Feature: Read ValueSet
-	When retrieving all value sets or searching value sets as an admin it should return
-	
-	
-		Scenario: As an admin I am able to retrieve all value sets
-			Given: I am an admin user 
-			when: I make a request to retrieve all value sets
-			then: Value sets should be returned
-			
-			
-		Scenario: As an admin I search for a value set that does not exist
-			Given: I am an admin user
-			when: I make a request for a value set that does not exist
-			then: A value sets should not be returned
-			
-			
-		Scenario: As an admin I search for a value set that exists
-			Given: I am an admin user
-			when: I search for a value set that exists
-			then: A value set should be returned
\ No newline at end of file
+@valueSet_search
+Feature: I can search for  value set
+
+
+    Scenario: I am able to retrieve all value sets
+            Given I am an admin user
+            When I make a request to retrieve all value sets
+            Then Value sets should be returned
+
+
+    Scenario: I can search for a value set that does not exist
+            Given I am an admin user
+            When I make a request for a value set that does not exist
+            Then A value sets should not be returned
+
+    Scenario: I can search for a value set that exists
+            Given I am an admin user
+            When I search for a value set that exists
+            Then Value sets should be returned
```

Comment:
Issue (blocking): Test should create its own data. Adding a `Background` that creates a value set would allow us to search for it specifically and verify that it is returned.